habitat-baselines/habitat_baselines/config/rearrange/ddppo_close_cab.yaml

```diff
@@ -1,83 +0,0 @@
-# @package _global_
```

Comment:
We will also need to remove the gym definition for those tasks then I guess

Reply:
It looks like all the simple rearrangement tasks were collapsed under [rl_skill.yaml](https://github.com/facebookresearch/habitat-lab/blob/hab2_fixup/habitat-baselines/habitat_baselines/config/rearrange/rl_skill.yaml). 

We should probably also modify [the docs](https://aihabitat.org/docs/habitat2/) indicating that rl skills can now be trained using the `rl_skill` and by modifying the [benchmark](https://github.com/facebookresearch/habitat-lab/blob/7da75b193c29814cfe10bb97c0db7d4172efb5f4/habitat-baselines/habitat_baselines/config/rearrange/rl_skill.yaml#L4) with hydra

Reply:
@ASzot Andrew, don't forget to update the [hab2 docs page](https://github.com/facebookresearch/habitat-lab/blob/main/docs/pages/habitat2.rst) (note, it now lives in the habitat-lab/docs not habitat-website).
I started doing that, but there were some changes e.g. VER is now default in `rl_skill`, + `hab` folder was removed from baselines config dir but there is HAB section in the docs. So maybe it would be better if you do it as a core Rearrangement contributor and domain expert.